src/utils/macAddress.ts

```diff
@@ -0,0 +1,21 @@
+import { useValidation, type ValidationAttrs } from '@/composable/validation';
+import type { Ref } from 'vue';
+
+function macAddressValidation(value: Ref): {
+  message: string;
+  status: 'error' | undefined;
+  isValid: boolean;
+  attrs: ValidationAttrs;
+} {
```

Comment:
No need for extra return types definition, let ts infer the correct type
```suggestion
function macAddressValidation(value: Ref): {
```